src/content/docs/accounts/install-new-relic/partner-based-installation/log-install-new-relic-partners.mdx

```diff
@@ -122,7 +122,7 @@ These partners require specific login and deployment procedures for you to use y
     2. Follow [New Relic's instructions](/docs/agents/php-agent/installation/wordpress-users-new-relic) to complete the installation process, or contact [W3TC support](http://wordpress.org/support/plugin/w3-total-cache) for assistance.
     3. Wait for New Relic to begin collecting data.
 
-       Within a few minutes after deploying the agent, performance data about your app will appear on APM's [Applications Overview p](/docs/apm/applications-menu/monitoring/applications-overview-dashboard)age.
+       Within a few minutes after deploying the agent, performance data about your app will appear on APM's [Applications Overview](/docs/apm/applications-menu/monitoring/applications-overview-dashboard) page.
```

Comment:
```suggestion
       Within a few minutes after deploying the agent, performance data about your app will appear on the APM [Applications Overview](/docs/apm/applications-menu/monitoring/applications-overview-dashboard) page.
```

Reply:
We don't make New Relic possessive, so thought we might avoid it with our capabilities. this is very nitpicky and not a blocker tho!

Reply:
Good catch! Modified!